Doc/library/doctest.rst

```diff
@@ -997,7 +997,7 @@ As your collection of doctest'ed modules grows, you'll want a way to run all
 their doctests systematically.  :mod:`doctest` provides two functions that can
 be used to create :mod:`unittest` test suites from modules and text files
 containing doctests.  To integrate with :mod:`unittest` test discovery, include
-a :func:`load_tests` function in your test module::
+a :func:`!load_tests` function in your test module::
```

Comment:
You could usefully link to https://docs.python.org/3/library/unittest.html#load-tests-protocol here

Reply:
Added!